src/styles/block/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+  background-color: var(--background-color);
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  align-self: center;
+}
+
+.card__productTitle {
+  line-height: 18px;
+  font-size: 12px;
+  margin-bottom: 4px;
+  font-weight: 500;
+  color: var(--mainColor);
+}
+
+.card__productCode {
+  line-height: 14px;
+  color: var(--text-color);
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 23px;
+}
+
+.card__rewiew {
+  font-weight: 400;
+  line-height: 14px;
+
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__priceText {
+  color: var(--text-color);
+  font-size: 12px;
+  font-weight: 400;
+}
+
+.card__priceValue {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--mainColor);
+}
+
+.card__button {
+  font-size: 14px;
+  font-weight: 700;
+  display: flex;
+  justify-items: center;
+  text-align: center;
+}
+
+.card__link {
+  color: var(--text-button-color);
+  border-radius: 5px;
+  text-transform: uppercase;
+  text-decoration: none;
+  line-height: 40px;
+  background-color: var(--button-color);
+  flex-grow: 1;
+  border: 1px solid var(--button-color);
+  transition: all 0.2s ease;
```

Comment:
don't use all with transitions, specify every property that changes to make it refactorable